.github/workflows/ci.yml

```
@@ -50,18 +50,20 @@ jobs:
       run: |
         echo "📊 Verifying test categorization by examining source files..."
         
-        # Count Category attributes in test files
+        # Count TestFixture and Test methods to ensure proper categorization
+        TESTFIXTURE_COUNT=$(find tests -name "*.fs" -exec grep -c "\[<TestFixture>\]" {} \; | awk '{sum += $1} END {print sum}')
         CATEGORY_COUNT=$(find tests -name "*.fs" -exec grep -c "\[<Category" {} \; | awk '{sum += $1} END {print sum}')
         TEST_COUNT=$(find tests -name "*.fs" -exec grep -c "\[<Test>" {} \; | awk '{sum += $1} END {print sum}')
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)